modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java

```diff
@@ -14,6 +14,8 @@
 
 package com.liferay.commerce.service.impl;
 
+import static com.liferay.commerce.constants.CommerceShipmentConstants.SHIPMENT_STATUS_PROCESSING;
```

Comment:
@gianmarcobrunialti we dont need to import this, since the constants class is already being imported below, line 20. We can just call that constant with `CommerceShipmentConstants.SHIPMENT_STATUS_PROCESSING;` where we need it to be used.

Reply:
Thanks for this, @dsitu . IntelliJ messed up with the import I think, didn't catch this.